internal/runtimehandlerhooks/default_cpu_load_balance_hooks.go

```diff
@@ -34,17 +33,16 @@ func (*DefaultCPULoadBalanceHooks) PostStop(ctx context.Context, c *oci.Containe
 	if node.CgroupIsV2() || c.Spoofed() {
 		return nil
 	}
-	containerCgroup, containerCgroupParent, systemd, err := containerCgroupAndParent(s.CgroupParent(), c)
+	_, containerManagers, err := libctrManagersForPodAndContainerCgroup(c, s.CgroupParent())
 	if err != nil {
 		return err
 	}
-	mgr, err := libctrManager(containerCgroup, containerCgroupParent, systemd)
-	if err != nil {
-		return err
-	}
-	path := mgr.Path("cpuset")
-	if path == "" {
-		return fmt.Errorf("failed to find cpuset for newly created cgroup")
+	for i := len(containerManagers) - 1; i >= 0; i-- {
```

Comment:
nonblocking: This block could be refactored to a function, since the same code is line high_performance_hooks.go:225

Reply:
done!